src/components/table/table.stories.js

```diff
@@ -35,24 +35,28 @@ const handleChange = (e, tableOptions) => {
   action('change')(e, tableOptions);
 };
 
+const recordsForActivePage = (start, end) => {
+  if (store.get('sortOrder') === 'desc' && store.get('sortedColumn').length) {
+    return countriesList.reverse().slice(start, end).toJS();
+  }
+  return countriesList.slice(start, end).toJS();
```

Comment:
I think it could be simplified:
```
  const records = countriesList;

  if (store.get('sortOrder') === 'desc' && store.get('sortedColumn').length) {
    records = records.reverse();
  }

  return records.slice(start, end).toJS();
```

Reply:
Yeah I was leaning towards reducing the complexity